### DISCUSSION TITLE
Can install on Ubuntu, but how about RHEL?

### DISCUSSION THREAD

**Discussion by Tridy** [strongswan/strongswan#565] [Q&A]
This [works on Ubuntu](https://docs.microsoft.com/en-us/azure/vpn-gateway/point-to-site-vpn-client-configuration-azure-cert#linuxinstallcli) to connect to Azure Network Gateway:
 
```
sudo apt install strongswan
sudo apt install strongswan-pki
sudo apt install libstrongswan-extra-plugins
```

how do I install that on Red Hat Enterprise Linux 7?

Thanks!

**Comment by Tridy** [comment]
no clues, I tried with **ogra**, not sure what it ts,

based on this : https://snapcraft.io/install/strongswan-ogra/rhel
I did this:
```
sudo rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
sudo yum update
sudo yum install snapd
sudo systemctl enable --now snapd.socket
sudo snap install strongswan-ogra --candidate
```
and then I get this:

```
[user@machine ~]$ ipsec
-bash: ipsec: command not found
```

**Reply by tobiasbrunner** [reply]
> no clues, I tried with **ogra**, not sure what it ts,

Seems to be a build from someone with the username _ogra_.

>     -bash: ipsec: command not found

The script might be renamed to _strongswan_ (as usually is the case with RPM packages).

**Comment by tobiasbrunner** [comment]
> how do I install that on Red Hat Enterprise Linux 7?

Probably via EPEL (see [InstallationDocumentation](https://wiki.strongswan.org/projects/strongswan/wiki/InstallationDocumentation)).

**Comment by Tridy** [comment]
I am using _Red Hat Enterprise Linux Server release 7.5 (Maipo)_ VM Image from Azure.
From what I see, to use it I will need EPEL and DNF dependencies, and I am getting the message:
```
This system is not registered with an entitlement server. You can use subscription-manager to register.
```

I am going to pass this task on to someone who can do magic in RHEL. My knowledge of Ubuntu is clearly not enough here.

**Reply by Thermi** [reply]
You need a license to get packages from the RedHat servers.

**Reply by Tridy** [reply]
That's what I sort of did not expect when paying for a VM instance of RHEL in Azure. I wanted a test environment where I can see how it works. Anyway, I should not be complaining about it here but blame MS and RH.

**Reply by Thermi** [reply]
Licensing and paying on Azure is a topic you need to investigate with them, yes. Some KB article on Azure probably explains it all.
